pkgs/applications/networking/instant-messengers/slack/default.nix

```diff
@@ -195,7 +194,10 @@ let
       runHook preInstall
       mkdir -p $out/Applications/Slack.app
       cp -R . $out/Applications/Slack.app
-      /usr/bin/defaults write com.tinyspeck.slackmacgap SlackNoAutoUpdates -bool YES
+    '' + lib.optionalString (!stdenv.isAarch64) ''
+      # on aarch64-darwin we get: Could not write domain com.tinyspeck.slackmacgap; exiting
+      /usr/bin/defaults write com.tinyspeck.slackmacgap SlackNoAutoUpdates -Bool YES
```

Comment:
https://github.com/hlolli/nixpkgs/blob/5facd430d4aedae75f8c4027fa62624225b9e752/pkgs/applications/networking/instant-messengers/slack/default.nix#L199
`-Bool` is wrong. `-bool` was correct.
When Intel Mac users are trying install Slack, it fails.

Reply:
> `-Bool` is wrong. `-bool` was correct.

Yup, I can run the command myself successfully when I use the correct casing `-bool`.
From the `installPhase`, though, it's still not happy.
> When Intel Mac users are trying install Slack, it fails.

Looks like the original issue was not limited to `aarch64-darwin` after all. Maybe something changed at the OS level. I'm having the same issue on `x86_64`.

Reply:
Same thing here, running `/usr/bin/defaults write com.tinyspeck.slackmacgap SlackNoAutoUpdates -bool YES` works in a terminal, but fails with `defaults[2342:1734766] Could not write domain com.tinyspeck.slackmacgap; exiting` when run from the `installPhase`.